app/controllers/discourse_automation/admin_discourse_automation_automations_controller.rb

```diff
@@ -3,7 +3,7 @@
 module DiscourseAutomation
   class AdminDiscourseAutomationAutomationsController < ::ApplicationController
     def index
-      automations = DiscourseAutomation::Automation.all
+      automations = DiscourseAutomation::Automation.order(updated_at: :desc).all
```

Comment:
Since we can't sort the list just yet I think ordering by name is more user friendly.

Reply:
sure makes sense :+1: